README.md

```diff
@@ -1,5 +1,130 @@
+
+### Project Goals
+
+For the purpose of this FT Global Legal Hackathon, this project addresses the following core components: 
+
+* Statistical process to reduce the identifiability of individuals while maintaining most of the usefulness of the data
+
+* Differentially private histogram approach for geospatial data which provides a theoretical guarantee to individual privacy.
+
+### Proposed Solution
+
+
+#### Project Assumptions & Out of Scope Matters
+
+Due to the shortened timeline of this project, our team made a number of assumptions and selected to only address a sub-set of the issues related to the trade-off between data location and privacy.
+
+1. Varying Legally Defined Terms 
+
+The key terms used in this project (see below) have a vareity of different defintions. Our project created standard defintions from those sources, but we recognize that these terms are arguablly nuanced. As such, in our Future Work, we will look to further explore these various defintions.
+
+2. INSERT
+
+### Legal Framework with Related Statistical Analysis 
+
+The use of location data as solution to address the spread of COVID-19 directly impacts interrelated but distinctly different terms under privacy law: 
+
+* Pseudonymization;
+* De-identification;
+* Anonymization; and
+* Aggregation.
+
+These terms are similar and often used interchangeably. The terms generally address the core idea of removing identifiers from data. However, with regulations and standards, each termcan be defined and used distinctinly differentlly. 
+
+#### Pseudonymization
+
+The term pseudonymization is most relevant within the European Union under the General Data Protection Regulation ("GDPR"). [1] The GDPR defines pseudonymization as: 
+
+> the processing of personal data in such a manner that the personal data can no longer be attributed to a specific data subject without the use of additional information, provided that such additional information is kept separately and is subject to technical and organisational measures to ensure that the personal data are not attributed to an identified or identifiable natural person [Art. 4(5)].
+
+Two key factors are inherent in this defition: (1) attribution and (2) additional information. Based on this defintion, it appears that so long as the "additional information" that can be use to identify the information is removed and separated from the non-identifer information, the data converts from personal data to data that does not fall under the GDPR.
+
+Based on this, our project generated the following generalized defintion for pseudonymization:
+
+> the processing of personal information in a manner that renders the personal information no longer attributable to a specific consumer without the use of additional information.
+
```

Comment:
Looks great.  should we put a "privacy strength" indicator somewhere.  Like Psuedonymization is the weakest because it is subject to data linkage attacks?

Reply:
I like the idea of identifying "Privacy Strength"